web-app/ormconfig.json

```diff
@@ -1,12 +1,11 @@
 {
-    "type": "postgres",
-    "host": "127.0.0.1",
-    "port": 5432,
-    "username": "postgres",
-    "password": "postgres",
-    "database": "test",
-    "entities": [
-        "dist/**/*.entity.js"
-    ],
-    "synchronize": true
-}
\ No newline at end of file
+  "type": "postgres",
+  "host": "127.0.0.1",
+  "port": 5432,
+  "username": "postgres",
+  "password": "postgres",
+  "database": "test",
+  "entities": ["dist/**/*.entity.js"],
+  "subscribers": ["dist/**/*.subscriber.js"],
```

Comment:
I fear you forgot to add this also to the Dockerfiles.

Example in Dockerfile.dev, you have a line like this: 

```
ENV TYPEORM_ENTITIES=dist/**/*.entity.js
```